packages/patternfly-4/react-core/CONTRIBUTING.md

```diff
@@ -7,102 +7,22 @@ section: patternfly-4
 ## Adding a new component
 
 1. Check for open issues that are not assigned to anyone, and assign yourself. If you do not see an issue for the component you want to contribute open an issue and assign yourself. Assigning yourself will ensure that others do not begin working on the component you currently have in progress.
-2. Generate the component scaffolding by running `yarn generate`. This will generate a structure that resembles the following
+1. Generate the component scaffolding by running `yarn generate`. This will generate a structure that resembles the following
    ```text
    packages/react-core/src/[type]/[ComponentName]/
-     index.js - Barrel File exporting public exports
-     ComponentName.js - Component Implementation
-     ComponentName.test.js - Component Tests
+     index.ts - Barrel File exporting public exports
+     ComponentName.tsx - Component Implementation
+     ComponentName.test.tsx - Component Tests
      ComponentName.md - Component Docs
      examples/ - dir for all examples
          SimpleComponentName.js - Simple Example
    ```
-3. Write the component implementation in `[Component].js`.
-4. Add jest tests to `[Component].test.js`. All new components must be tested.
-5. Add any additional public exports to `index.js`
-6. Update the generated `[ComponentName].md.` See how to create [component docs.](../react-core/README.md)
-7. Add integration tests to the demo-app found [here](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-integration)
-
-
-## Code contribution guidelines
-
-Adhering to the following process is the best way to get your work included in the project:
-
-1.  [Fork](https://help.github.com/fork-a-repo/) the project, clone your fork, and configure the remotes:
-
-```bash
-# Clone your fork of the repo into the current directory
-git clone https://github.com/<your-username>/patternfly-react.git
-# Navigate to the newly cloned directory
-cd patternfly-react
-# Assign the original repo to a remote called "upstream"
-git remote add upstream https://github.com/patternfly/patternfly-react.git
-```
-
-2.  Create a branch:
-
-```text
-$ git checkout -b my-branch -t upstream/master
-```
-
-3. Generate your component
-
-```bash
-# Run the tool to Generate the component scaffolding
- yarn generate
-```
-
-- When you select the option to generate a PatternFly 4 component, a structure resembling the following is generated
-  ```text
-  packages/patternfly-4/react-core/src/[type]/[ComponentName]/
-    index.js - Barrel File exporting public exports
-    ComponentName.js - Component Implementation
-    ComponentName.test.js - Component Tests
-    ComponentName.md - Component Docs
-  ```
-
-4.  Develop your component. After development is complete, ensure tests and lint standards pass.
```

Comment:
can you add these steps back in.  I think they are important for contributors.

Reply:
See previous reasons for PF3's CONTRIBUTING.md